projects/demo/src/modules/markup/icons/customization/customization-icons.component.ts

```diff
@@ -15,15 +16,19 @@ import {NgDompurifySanitizer} from '@tinkoff/ng-dompurify';
             provide: TUI_SANITIZER,
             useClass: NgDompurifySanitizer,
         },
-        tuiSvgOptionsProvider({
-            srcProcessor: src => {
-                const myCustomPrefix = 'icons8::';
-
-                return String(src).startsWith(myCustomPrefix)
-                    ? `assets/icons8/${String(src).replace(myCustomPrefix, '')}.svg`
-                    : src;
-            },
-        }),
+        /**
+         * @note:
+         * Be careful, component has its own injector which doesn't
+         * inherit providers from parent injector. If you want to keep
+         * the module's injectors, then use the provider in the module,
```

Comment:
It would also lose global interceptors if it's a lazy module. Better to write something like "It's best to keep it global and provide all necessary interceptors at the top level of your app".